bin/filter_sce_rds.R

```diff
@@ -62,14 +74,18 @@ filtered_sce <- filtered_sce |>
 # since this can fail, we will check for success
 miQC_worked <- FALSE
 try({
-  filtered_sce <- scpcaTools::add_miQC(filtered_sce)
+  filtered_sce <- scpcaTools::add_miQC(filtered_sce,
+                                       posterior_cutoff = opt$prob_compromised_cutoff)
+  metadata(filtered_sce)$prob_compromised_cutoff <- opt$prob_compromised_cutoff
   miQC_worked <- TRUE
  })
  
 # set prob_compromised to NA if miQC failed
 if (!miQC_worked){
   warning("miQC failed. Setting `prob_compromised` to NA.")
   filtered_sce$prob_compromised <- NA_real_
+  filtered_sce$miQC_pass <- NA_real_
```

Comment:
Since this is a logical value, NA should be fine on its own.
```suggestion
  filtered_sce$miQC_pass <- NA
```